html/browsers/history/the-history-interface/history_go_to_uri-1.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<script src="history.js"></script>
+<script>
+  onunload = function() {}
+
+  onload = function() {
+    if (!opener.started) {
+      queue_next();
+    } else {
+      opener.pages.push(id);
+      opener.start_test_wait();
+      if (!opener.gone) {
+          history.go("history_entry.html");
```

Comment:
yes

On Jul 29, 2016 3:03 PM, "John Jansen" <notifications@github.com> wrote:

> In html/browsers/history/the-history-interface/history_go_to_uri-1.html
> <https://github.com/w3c/web-platform-tests/pull/3306#discussion_r72844681>
> :
>
> > @@ -0,0 +1,18 @@
> > +<!doctype html>
> > +<script src="history.js"></script>
> > +<script>
> > +  onunload = function() {}
> > +
> > +  onload = function() {
> > +    if (!opener.started) {
> > +      queue_next();
> > +    } else {
> > +      opener.pages.push(id);
> > +      opener.start_test_wait();
> > +      if (!opener.gone) {
> > +          history.go("history_entry.html");
>
> I fixed the build issue. You cool with merging?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/web-platform-tests/pull/3306/files/5c17662599d62fe174c3984841c70fadbbfe4fa4#r72844681>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH8lSTsCDpgb11H2w-ba9dKB30K0sn8uks5qak5ygaJpZM4JO7lz>
> .
>